platform-coop-toolkit/pinecone
# fix: call-to-action, accordion, tags, and filter list state styling

## Description
This PR addresses a number of styling issues:

* [x] Call to action text not readable when inverted (Fixes #359 )
* [x] Accordion does not use the proper focus and active styles (Fixes #366 )
* [x] Tag missing active state (Fixes #362 )
* [x] Tag focus border is partly too thin (Fixes #316)

## Steps to test

1. Go to the deployed instance for this PR: https://deploy-preview-380--pinecone.netlify.app/
2. Examine:
   * Atom > Tag
   * Molecule > Accordion
   * Molecule > Filter List
   * Layout > Search Results (check Filter list)
   * Layout > Archive (check Filter list)
   * Layout > Resource (check Tags)

**Expected behavior:**

The focus, hover, and active states should look like the design spec in major current browsers.

## Related issues
Fixes #359 
Fixes #366 
Fixes #362 
Fixes #316
